### PR TITLE
Add Documentation for LSTM examples

### PR DESCRIPTION
**Pull request by kartikdutt18**
There are some style fixes that are left to do and I also need to add an index but I think this is what should be covered in the readme. @codeboy5, @favre49 Let me know what you think?

### CODE REVIEW

File: LSTM/README.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+A sample from the dataset is shown below. Here only `Consumption` is dependent on time hence this is a univariate dataset.
+
+|DateTime|Consumption kWh|Off-peak|Mid-peak|On-peak |
+|--------|---------------|--------|--------|--------|
+|11/25/2011 01:00:00|0.39           |1       |0       |0       |
+|11/25/2011 02:00:00|0.33           |1       |0       |0       |
+|11/25/2011 03:00:00|0.27           |1       |0       |0       |
+
+Getting Started :
```
[the remark this answers is not in the document]
Comment (kartikdutt18) [reply LSTM/README.md]: Yes, This was just to show the overall idea of what I had in mind that should be included. Sorry you had to go through this without the proper headings and style fixes. Do you think a documentation should look like this?

File: LSTM/README.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+```
+
+b) We set input and output size for model.
+
+For Univariate the input and output size will always be 1. However for multivariate we will taking input all columns and predicting high and low.
+Hence for multivariate dataset we will use : `size_t inputSize = 5, outputSize = 2;`
+
+c) Splitting the data into training and validation sets.
+
+We avoid using internal Split function to avoid shuffling.
```
[the remark this answers is not in the document]
Comment (kartikdutt18) [reply LSTM/README.md]: Ahh got it. Will add it in the next commit.

File: LSTM/README.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+```
+arma::mat trainData = dataset.submat(arma::span(),arma::span(0, (1 - RATIO) *
+      dataset.n_cols));
+arma::mat testData = dataset.submat(arma::span(), arma::span((1 - RATIO) * dataset.n_cols,
+      dataset.n_cols - 1));
+```
+
+d) Scale the dataset.
+
+mlpack has an excellent support for multiple scalers, here we will be using MinMaxScaler.
```
[the remark this answers is not in the document]
Comment (kartikdutt18) [reply LSTM/README.md]: Will do.

File: LSTM/ReadMe.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
+Time Series Forecasting
```
[the remark this answers is not in the document]
Comment (kartikdutt18) [reply LSTM/ReadMe.md]: I don't know what happened here. I was using my previous fork and I hadn't changed the origin when I created this, Once I did and rebased I got two copies on the remote branch but only a single copy locally.

File: LSTM/README.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+        STEP_SIZE, // Step size of the optimizer.
+        BATCH_SIZE, // Batch size. Number of data points that are used in each iteration.
+        trainData.n_cols * EPOCHS, // Max number of iterations.
+        1e-8,// Tolerance.
+        true, // Shuffle.
+        AdamUpdate(1e-8, 0.9, 0.999)); // Adam update policy.
+```
+
+We can simply train the model as shown below:
+Ensmallen callbacks are useful for optimization, printing progress as well as stoping at minimum loss as shown below.
```
[the remark this answers is not in the document]
Comment (kartikdutt18) [reply LSTM/README.md]: Agreed, will do.

**Comment by codeboy5** [comment]
> There are some style fixes that are left to do and I also need to add an index but I think this is what should be covered in the readme. @codeboy5, @favre49 Let me know what you think?

Seems okay to me, will go through the final version once.
I will change mine to a similar format.

File: LSTM/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (favre49) [reply LSTM/README.md]: Yup, it looks good. Just needs some refining.

**Comment by rcurtin** [comment]
Just an overall comment---I think it would be really nice to instead remove this `README` file, replace it with only a very short description of the task ('predict Google stock price using mlpack LSTMs'), and then work most of the comments here into the notebook in #72 or a .cpp file in the directory (or both).  Let me know what you think; just an idea to reduce duplication. :+1:

**Comment by kartikdutt18** [comment]
Agreed, that makes more sense. I am closing this and will be shifted in #72.

**Comment by rcurtin** [comment]
Sounds good, thanks! :+1:
